domain/schema/model/sql/0020-network.sql

```diff
@@ -23,3 +23,189 @@ CREATE TABLE cloud_container (
     FOREIGN KEY (net_node_uuid)
     REFERENCES net_node (uuid)
 );
+
+CREATE TABLE cloud_container_ip_address (
```

Comment:
This is going back to what we had, which is what don't want.

There shall be no different path for addressing across machines, cloud containers and cloud services.

All addresses will be against link-layer devices, which will need a new table. For cases where the the device is _unknown_, we can use that type, or a new type for a pass-through/place-holder device.

I believe this is described in the Jira card.

Reply:
It's described in the card but clearly I didn't grok the intent since I couldn't see how a pod (which is the physical counterpart to the cloud container logical entity) gets a link layer device when that's not something that is part of the k8s model. Happy to have some input to get this sorted.